src/main/webapp/registration.html

```diff
@@ -75,7 +75,7 @@
       
         <div class="bg-description-blue">
             <div class="container-fluid bg-white">
-                <form id="registration-form" action="/registration" method="POST" style="display:none">
+                <form id="registration-form" enctype="multipart/form-data" method="POST" style="display:none">
```

Comment:
What does this do?

Reply:
multipart/form-data is used for encoding data that involves a file input. Here's a good thread explaining it in more detail
https://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean